### PR TITLE
replaced bootstrap with scss

### PR DESCRIPTION
**Pull request by DanielCadiz**
## The Story
-removed all bootstrap on partials: footer, footer2 and cardbuild
-added scss for their styling

## Screenshots
scss_footer_screenshot

### CODE REVIEW

File: app/views/shared/_footer2.html.erb [hunk 1]
```diff
@@ -1,42 +1,41 @@
...
+        <li><a class="info_text" href="#">Build a PC</a></li>
+        <li><a class="info_text" href="#">Build Guide</a></li>
+        <li><a class="info_text" href="#">Builds Gallery</a></li>
+        <li><a class="info_text" href="#">Browse Products</a></li>
       </ul>
     </div>
 
     <div class="info">
-      <h5>INFORMARION</h5>
+      <h3>INFORMARION</h3>
```
Comment (AdrianIlaga) [inline app/views/shared/_footer2.html.erb]: Typo on "INFORMATION"

**Review by AdrianIlaga — COMMENTED** [review]
Mostly good. Spotted a typo on _footer2.html.erb line 24

File: app/views/shared/_footer2.html.erb (hunk 1 above)
Comment (DanielCadiz) [reply app/views/shared/_footer2.html.erb]: alright edited the commit of _footer2.html.erb to correct the spelling error.

**Comment by jarencudilla** [comment]
Thank you for catching the typo.

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -12,9 +12,9 @@
         <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum volutpat laoreet risus sed condimentum. Nulla facilisi. Nunc mattis purus in bibendum blandit. Phasellus fermentum erat ultrices nunc aliquam interdum. Mauris rhoncus, nisi quis scelerisque dapibus, lorem ex fringilla felis, et mattis nisl eros at massa. Aenean condimentum tempus ornare. Ut scelerisque nulla ut elit iaculis, nec egestas turpis tincidunt. Suspendisse elementum pharetra auctor. Donec semper imperdiet nisi, sit amet varius sem commodo sed. Pellentesque bibendum euismod risus, luctus iaculis lacus hendrerit dictum. Proin at lectus pharetra, pharetra quam quis, dapibus purus.</p>
       </div>
     </div>
+    <%= render "shared/footer", class: "foot" %>
   <% else %>
     <%= render "shared/cardbuild", class: "cardbuild" %>
-    <%#= render "shared/footer", class: "foot" %>
     <%= render "shared/footer2", class: "foot" %>
   <% end %>
```
Comment (jarencudilla) [inline app/views/home/index.html.erb:20]: Nice, This is what I'm talking about. We can isolate everything into partials so it's easier to check and update if necessary!

**Pull request merged**